internal/common/cloud/providers.go

```diff
@@ -0,0 +1,22 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloud
+
+const (
+	// ProviderAWS is used in cloud.provider label for AWS.
+	ProviderAWS = "aws"
+	// ProviderGCP is used in cloud.provider label for GCP.
+	ProviderGCP = "gcp"
```

Comment:
Should this be in core semantic conventions or this is Splunk convention only?

Reply:
aws and gcp are called out as examples in the conventions (https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/resource/semantic_conventions/cloud.md). Should I create a PR to amend semantic conventions to make it explicit that these values should be used for the platforms? If so I think then it'd make sense to put this in translator/conventions/opentelemetry.go.

Reply:
Yes, best to add to core conventions.